classification_banner/banner.py

```diff
@@ -49,9 +49,9 @@ def get_host():
 def configure():
     """Read Global configuration"""
     defaults = {}
-    defaults["message"] = "UNCLASSIFIED"
-    defaults["foreground"] = "#FFFFFF"
-    defaults["background"] = "#007A33"
+    defaults["message"] = "NOT CONFIGURED"
```

Comment:
Not sure how most users would react to a banner on the screen that says ``NOT CONFIGURED``. It's unclear what isn't configured.

Perhaps something more explicit, such as ``SYSTEM CLASSIFICATION NOT CONFIGURED`` would be more useful?

Reply:
p.s. regardless of wording, definitely agree on the intent of ``NOT CONFIGURED``

Reply:
Thanks. My suggestion is following windows netbanner default. A user should never see not configured. A system admin should, and configure it accordingly.

Reply:
Sorry it took so long to merge.